binutils/tests/test.bats

```diff
@@ -0,0 +1,33 @@
+@test "ld.bfd sets -rpath" {
+  # Set up our LD_RUN_PATH to simulate what plan-build would do.
+  # Use multiple entries to ensure splitting is performed correctly
+  LD_RUN_PATH="/foo/bar:/baz/lib"
+
+  # hab pkg exec will unset LD_RUN_PATH, so we need to call the full
+  # path in this case.
+  DEBUG=true LD_RUN_PATH=$LD_RUN_PATH run /hab/pkgs/$TEST_PKG_IDENT/bin/ld.bfd
```

Comment:
Is it intentional not to use `hab pkg exec` here? If so, the comment above needs adjusted.

Reply:
I am not reading properly, apologies.